includes/grids-toolbar.md

```diff
@@ -0,0 +1,264 @@
+Predefined controls appear on the toolbar depending on whether a specific DataGrid feature is enabled. The following table illustrates the dependency:
+
+## Predefined Controls
+Predefined controls include:
+
+<table class="dx-table">
+	<tr>
+		<th>Control Name</th>
+		<th>Image</th>
+		<th>Prerequisites</th>
+	</tr>
+	<tr>
+		<td><i>addRowButton</i></td>
+		<td><img src="/images/DataGrid/icons/toolbar_add.png" alt="DevExtreme JavaScript jQuery Angular UI component {WidgetName} Toolbar AddButton" style="margin:1px; vertical-align:middle"/></td>
+		<td><a href="{basewidgetpath}/Configuration/editing/">editing</a>.<a href="{basewidgetpath}/Configuration/editing/#allowAdding">allowAdding</a> is set to <b>true</b></td>
+	</tr>
+	<tr>
+		<td><i>applyFilterButton</i></td>
+		<td><img src="/images/DataGrid/icons/toolbar_applyFilter.png" alt="DevExtreme JavaScript jQuery Angular UI component {WidgetName} Toolbar ApplyFilterButton" style="margin:1px; vertical-align:middle"/></td>
+		<td><a href="{basewidgetpath}/Configuration/filterRow/">filterRow</a>.<a href="{basewidgetpath}/Configuration/filterRow/#visible">visible</a> is set to <b>true</b> and filterRow.<a href="{basewidgetpath}/Configuration/filterRow/#applyFilter">applyFilter</a> is set to <i>"onClick"</i></td>
+	</tr>
+	<tr>
+		<td><i>columnChooserButton</i></td>
+		<td><img src="/images/DataGrid/icons/columnChooser.png" alt="DevExtreme JavaScript jQuery Angular UI component {WidgetName} Toolbar ColumnChooserButton" style="margin:1px; vertical-align:middle"/></td>
+		<td><a href="{basewidgetpath}/Configuration/columnChooser/">columnChooser</a>.<a href="{basewidgetpath}/Configuration/columnChooser/#enabled">enabled</a> is set to <b>true</b></td>
+	</tr>
+	<tr>
+		<td><i>revertButton</i></td>
+		<td><img src="/images/DataGrid/icons/toolbar_revert.png" alt="DevExtreme JavaScript jQuery Angular UI component {WidgetName} Toolbar RevertButton" style="margin:1px; vertical-align:middle"/></td>
+		<td><a href="{basewidgetpath}/Configuration/editing/">editing</a>.<a href="{basewidgetpath}/Configuration/editing/#mode">mode</a> is set to <i>"batch"</i></td>
+	</tr>
+	<tr>
+		<td><i>saveButton</i></td>
+		<td><img src="/images/DataGrid/icons/toolbar_save.png" alt="DevExtreme JavaScript jQuery Angular UI component {WidgetName} Toolbar SaveButton" style="margin:1px; vertical-align:middle"/></td>
+		<td><a href="{basewidgetpath}/Configuration/editing/">editing</a>.<a href="{basewidgetpath}/Configuration/editing/#mode">mode</a> is set to <i>"batch"</i> and <a href="{basewidgetpath}/Configuration/editing/">editing</a>.<a href="{basewidgetpath}/Configuration/editing/#allowUpdating">allowUpdating</a> is <b>true</b></td>
+	</tr>
+	<tr>
+		<td><i>searchPanel</i></td>
+		<td><img src="/images/DataGrid/icons/toolbar_searchPanel.png" alt="DevExtreme JavaScript jQuery Angular UI component {WidgetName} Toolbar searchPanel" style="margin:1px; vertical-align:middle"/></td>
```

Comment:
Cut the white borders:

![image](https://user-images.githubusercontent.com/18551316/131869781-47054fd9-34ba-42f1-ac5b-e88a4299650a.png)